test/common/ops/revive.js

```diff
@@ -65,15 +79,55 @@ describe('shared.ops.revive', () => {
     expect(user.items.gear.owned[weaponKey]).to.be.false;
   });
 
-  it('does not remove 0 value items');
+  it('does not remove 0 value items', () => {
+    let eyewearKey = 'eyewear_special_yellowTopFrame';
+    user.items.gear.owned[eyewearKey] = true;
```

Comment:
To make sure that this is the only owned item, let's do:

```js
user.items.gear.owned = {
  eyewear_special_yellowTopFrame: true, // eslint-disable-line camelcase
};
```

It's okay to use the full key name when setting it up and making the assertion. Not DRY, but it makes the test easier to read.